### PR TITLE
Add feature gate InformerResourceVersion for v1.30

### PR DESCRIPTION
**Pull request by tengqm**

### CODE REVIEW

File: content/en/docs/reference/command-line-tools-reference/feature-gates/informer-resource-version.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+_build:
+  list: never
+  render: false
+
+stages:
+  - stage: alpha 
+    defaultValue: false
+    fromVersion: "1.30"
+---
+Checks the last synced resource version using the informer.
```
Comment (sftim) [inline content/en/docs/reference/command-line-tools-reference/feature-gates/informer-resource-version.md:13]: I'm worried people won't know what this means.

**Review by sftim — COMMENTED** [review]
LGTM for docs

Which SIG owns the related code change?

**Comment by drewhagen** [comment]
- Searched for [`InformerResourceVersion` in Kubernetes code base](https://github.com/search?q=repo%3Akubernetes%2Fkubernetes%20InformerResourceVersion&type=code)
- Tracked to this [code PR](https://github.com/kubernetes/kubernetes/commit/9161302e7fd3a5fb055b2f2572c6e1228240bb51#diff-b7315e8b207a5ae5ec05d07eb589aead09d071b9b29cf9e4b82d469b234bbb0cR43).
- This was implemented for [KEP-4192](https://github.com/kubernetes/enhancements/issues/4192).

/sig api-machinery

cc: @nilekhc @deads2k @liggitt

Can you please help contribute some detail to this feature gate doc? We appreciate it, thanks!

**Comment by T-Lakshmi** [comment]
> 	InformerResourceVersion provides the resource version observed when last synced with the underlying store. The value returned is not synchronized with access to the underlying store and is not thread-safe.

As per the comments in [code](https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/client-go/tools/cache/shared_informer.go#L182).

**Pull request closed without merging**